views/js/qtiCreator/widgets/interactions/matchInteraction/states/Map.js

```diff
@@ -17,30 +17,35 @@ define([
 
         //finally, apply defined correct response and response mapping:
         responseWidget.setResponse(interaction, _.values(response.getCorrect()));
-        
-       //change the correct state 
-       _widget.on('metaChange', function(meta){
+
+        $('.match-interaction-area input[type="checkbox"]', _widget.$container)
```

Comment:
As `$('.match-interaction-area input[type="checkbox"]', _widget.$container)` is a recurrent selector on your code I would use a constant instead.

`const interaction = $('.match-interaction-area input[type="checkbox"]', _widget.$container);`

Reply:
Done :)